### PR TITLE
Rough command for importing organizations into the website

### PR DESCRIPTION
**Pull request by ronaldheft-usds**
There is a lot to be desired in this script, but it _should_ handle our import for this one off case.

This PR will **not**  be merged and is for reference only.

### CODE REVIEW

**Comment by nickrobison-usds** [comment]
Before we go too far down this road, I just want to ask one dumb question. Is this a process that we will need to replicate for our production pilots as well when we go live?

If so, does it make more sense to implement #447 so we have an existing solution for that time?

**Comment by ronaldheft-usds** [comment]
@nickrobison-usds We won't. The website will function as expected creating their tokens in production because the website will already have its internal organization created and will be then responsible for creating the org in production.

File: dpc-attribution/src/main/java/gov/cms/dpc/attribution/cli/ImportCommand.java [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+            String name = resultSet.getString(3);
+            String endpointId = resultSet.getString(4);
+
+            this.importIntoWebsite(webConnection, id, npi, name, endpointId);
+        }
+    }
+
+    protected void importIntoWebsite(Connection webConnection, String id, String npi, String name, String endpointId) throws Exception {
+        String orgInsert = String.format(
+                "INSERT INTO organizations(name, npi, organization_type, num_providers, api_environments, created_at, updated_at) VALUES ('%s', '%s', 0, 0, '{0}', now(), now())",
```
Comment (switzersc-usds) [inline dpc-attribution/src/main/java/gov/cms/dpc/attribution/cli/ImportCommand.java]: just wanna note that the org name should be unique. is that enforced in the api at all or do we risk inserting non-unique names into the website db?

File: dpc-attribution/src/main/java/gov/cms/dpc/attribution/cli/ImportCommand.java [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+        );
+        System.out.println(fhirInsert);
+
+        if ( RUN_INSERTS ) {
+            PreparedStatement fhirInsertStatement = webConnection.prepareStatement(fhirInsert);
+            fhirInsertStatement.executeUpdate();
+        }
+
+        String addressInsert = String.format(
+                "INSERT INTO addresses(street, city, state, zip, addressable_type, addressable_id, created_at, updated_at, address_use, address_type) VALUES ('123 Fake St.', 'Washington', 'DC', '20000', 'Organization', %s, now(), now(), 0, 0)",
```
Comment (switzersc-usds) [inline dpc-attribution/src/main/java/gov/cms/dpc/attribution/cli/ImportCommand.java]: Lol okay

File: dpc-attribution/src/main/java/gov/cms/dpc/attribution/cli/ImportCommand.java (hunk 1 above)
Comment (ronaldheft-usds) [reply dpc-attribution/src/main/java/gov/cms/dpc/attribution/cli/ImportCommand.java]: It's not enforced in the API, but it appears the website is okay with duplicate names. I was able to import the same org multiple times in DEV.

Comment (switzersc-usds) [reply dpc-attribution/src/main/java/gov/cms/dpc/attribution/cli/ImportCommand.java]: It's okay at the database level but at the model level those records will be considered invalid, so a user will not be able to update them until the name is changed to be unique.

**Comment by ronaldheft-usds** [comment]
This was run successfully.

**Pull request closed without merging**